### ISSUE TITLE
FDSN webservice uncertainty unit

### ISSUE THREAD

**Issue by Brtle** [SeisComP3/seiscomp3#181]
The FDSN webservice generates QuakeML with latitude and longitude uncertainties in km. However, the [QuakeML latitude and longitude are in degrees](https://quake.ethz.ch/quakeml/docs/REC?action=AttachFile&do=get&target=QuakeML-BED-20130214a.pdf#page=30) and we can therefore think that the uncertainty should also be in degrees.

This unit conversion issue also exists in the XSLT files but it's much harder to fix. The conversion from km to degrees involves some cosine computation that isn't available in the XSLT stylesheet. I don't know how we can do this properly.

**Comment by jordi-domingo** [comment]
Agree, it is confusing. It would be nice that either QuakeML specifies that latitude / longitude uncertainties are in km, or that SC3's fdsnws does the conversion to degrees.
